consensus/README.md

```diff
@@ -11,7 +11,6 @@ be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 ## Notation
 
 - `x || y`, x concatenated with y
-- `sk <- rand(1^n)`, sample a random n bit number
```

Comment:
Initially I wanted to change it to `rand(2^n)` but didn't able to find it usages